migrations/founders.ganache.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
why need this file ?
why need ganache in the file name ?
why not just have founders.json for both ganache and kovan ?

Reply:
merged both into founders.json